### PR TITLE
feat: implement miam info screen (PRLC100-173)

### PR DESCRIPTION
**Pull request by rahulsanjava19**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/miam-info/content.test.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import { FormContent, FormFields, FormOptions, LanguageLookup } from '../../../../app/form/Form';
+import { CommonContent, generatePageContent } from '../../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../../app/form/validation');
+
+const en = {
+  serviceName: 'Child Arrangements',
+  headingTitle: 'Attending a Mediation Information and Assessment Meeting MIAM',
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/miam/miam-info/content.test.ts:11]: Change this to title

File: src/main/steps/c100-rebuild/miam/miam-info/content.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  serviceName: 'Child Arrangements',
+  headingTitle: 'Attending a Mediation Information and Assessment Meeting MIAM',
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/miam/miam-info/content.ts:8]: Change this to title

File: src/main/steps/c100-rebuild/miam/miam-info/content.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    miamConsent: {
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/miam/miam-info/content.ts:100]: you will have to rename the fields to miam_

Refer to this PR - https://github.com/hmcts/prl-citizen-frontend/pull/351/files

File: src/main/steps/c100-rebuild/miam/miam-info/template.njk [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{% extends "common/template.njk" %}
+
+
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/miam/miam-info/template.njk:3]: please check if we can add all these as part of content.ts

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/app/case/case.ts [hunk 5]
```diff
@@ -64,10 +64,12 @@ export interface Case {
   caseCode?: string;
   accessCode?: string;
   detailsKnown?: string;
+  miam_otherProceedings?: string;
   miam_haveDocSigned?: string;
   startAlternative?: string;
   contactDetailsPrivate?: string;
   miamStart?: string;
+  miamConsent?: string;
```
Comment (channa-patil) [inline src/main/app/case/case.ts:72]: @rahulsanjava19 Please rename this field to 'miam_consent'
If this is existing field then define a new one & don't alter the existing.

File: src/main/steps/c100-rebuild/miam/miam-info/content.ts [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  legalAidLink:
+    '<a href="https://www.gov.uk/check-legal-aid" class="govuk-link" target="_blank" aria-label="Check if you’re eligible for legal aid during mediation">Check if you’re eligible for legal aid during mediation</a> - welsh',
+  miamMeeting:
+    'A <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> is a one-off meeting and not the same as mediation. - welsh',
+  miamHeading1: 'What happens at a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading2:
+    'If you’ve already attended a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading3:
+    'When you don’t have to attend a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamList1: 'how mediation works - welsh',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/miam-info/content.ts:64]: @rahulsanjava19 We can define this as an Array & use a for loop in template.njk to render.
Please refer starting guidance page or safety concerns guidance.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/miam-info/template.njk (hunk 4 above)
Comment (rahulsanjava19) [reply src/main/steps/c100-rebuild/miam/miam-info/template.njk:3]: These all fields are as part of content.ts. it's been just referred here

**Review by channa-patil — APPROVED** [review]

**Pull request merged**